Vikta-API-testing-sample/src/test/java/com/test/api/category/CategoryTest.java

```diff
@@ -20,6 +22,7 @@
      */
     @Test
     public void verifyGetCategory() {
+
         final int categoryId = 5;
         final String expectedTitle = "Street";
 
```

Comment:
I know it's not the message You set, but please rework all the assertions messages in following manner:
"Category title should be %s, but is %s"
Using this approach gives a clearer understanding of what is expected and what is the actual outcome when assertion fails. Please apply to all assertions